src/index.js

```diff
@@ -137,13 +146,30 @@ async function run(params) {
   params.provider = secondLvlDom;
 
   const urlString = url.toString();
-  const result = await embed(urlString, params);
 
-  if ((params.api || params.OEMBED_RESOLVER_URI) && result.headers['X-Provider'] !== 'Helix') {
-    // filter all __ow_something parameters out
-    return serviceembed(params, urlString, log);
+  // check that OEMBED_RESOLVER_URI and a OEMBED service key are included
+  if ((params.api || params.OEMBED_RESOLVER_URI)) {
+    // add service based embed to be concurrently resolved
+    promises.push(serviceembed(params, urlString, log));
   }
-  return result;
+  promises.push(embed(urlString, params));
```

Comment:
if you push this first, you know the 2nd might be optional. this makes the test below easier...